.github/workflows/deploy-server.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to Amazon ECS
+
+on:
+  pull_request: # for now, trigger this on PR so I can test it before merging
+
+env:
+  AWS_REGION: us-east-2
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: docker/build-push-action@v1
+        name: Build and Push Docker image
+        with:
+          repository: joshprzybyszewski/cribbage
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          tag_with_ref: true
+          add_git_labels: true
+          tag_with_sha: false
```

Comment:
We'd probably want to start doing this soon so we can deploy different versions if needed :man_shrugging: